src/persistence/db/upgrades/dbupgrader.cpp

```diff
@@ -285,13 +287,15 @@ bool DbUpgrader::createCurrentSchema(RawDatabase& db)
 {
     QVector<RawDatabase::Query> queries;
     queries += RawDatabase::Query(QStringLiteral(
-        "CREATE TABLE peers (id INTEGER PRIMARY KEY, "
-        "public_key TEXT NOT NULL UNIQUE);"
+        "CREATE TABLE authors (id INTEGER PRIMARY KEY, "
+        "public_key BLOB NOT NULL UNIQUE);"
+        "CREATE TABLE chats (id INTEGER PRIMARY KEY, "
+        "uuid BLOB NOT NULL UNIQUE);"
```

Comment:
I guess it makes sense to do this rename now since we won't need to rename again on group history implementation, but I will note that this confused me a little. "Why is this called a uuid? It's always a friend pk".

Reply:
Yeah fair.. just trying to save on the extra upgrade for group history where group id's are just documented as "IDs".